### PR TITLE
Web portal

### PR DESCRIPTION
**Pull request by Guptavivek27**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,14 +10,17 @@ Interactive Visualization Platform for Machine Learning and Data Science Dataset
...
 ## Requirements and Setup
 
-In order to run the Neurobazaar Platform you need to have Python and the GCC compiler installed on your computer.
+In order to run the Neurobazaar Platform you need to have Python **3.11** and Node.js **v22.5** installed on your computer.
 
-This software uses VTK version 9.3.1.
+This software uses VTK version **9.3.1**.
 
-This software has been developed and tested with Python **3.11** and GCC **12.3** on Ubuntu 22.04 LTS Server Edition.
+This software has been developed and tested on Ubuntu **22.04 LTS**.
```
Comment (alexandru-orhean) [inline README.md]: You need to add information about the GCC compiler as well.

File: README.md [hunk 2]
```diff
@@ -10,14 +10,17 @@ Interactive Visualization Platform for Machine Learning and Data Science Dataset
...
 
 ### Ubuntu Required Packages
 
+To install and manage Node.js installations you can use Node Version Manager. Use the following link to install the latest version of NVM:
+https://github.com/nvm-sh/nvm
```
Comment (alexandru-orhean) [inline README.md:20]: This is not a Ubuntu dependency. Please include the Node.js instructions in a different section. The instructions should be placed after install the Ubuntu dependencies.

File: README.md [hunk 3]
```diff
@@ -36,30 +39,23 @@ sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-12 120
...
 All of the required Python packages are installed in a Python Virtual Environment. The first step is to create the virtual environment and the second step is to load/activate the environment. The first step has to be run only once.
 
-Create the Python Virtual Environment:  
+Create the Python Virtual Environment on Ubuntu 22.04 LTS:  
 ```
 python3.11 -m venv .venv
 ```
 
-Load/activate the Python Virtual Environment: 
+Load/activate the Python Virtual Environment in Ubuntu 22.04 LTS:
```
Comment (alexandru-orhean) [inline README.md]: We already mentioned that we only support Ubuntu 22.04 LTS Server Edition. Mentioning it again is not required.

File: README.md [hunk 4]
```diff
@@ -36,30 +39,23 @@ sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-12 120
...
 Before you can build VTK, you need to initalize the VTK submodule.
 
 ```
 git submodule init
 git submodule update
-```
-
-Compile and install the VTK modules:
-
-```
```
Comment (alexandru-orhean) [inline README.md]: The readme from the experimental branch is more up to date. Please include those instructions.

**Review by alexandru-orhean — CHANGES_REQUESTED** [review]
Please update the readme so that it's more in line with the readme from the experimental branch.

**Comment by Guptavivek27** [comment]
> Please update the readme so that it's more in line with the readme from the experimental branch.

Updates done.  Please proceed with the merge.

**Review by alexandru-orhean — APPROVED** [review]
Looks good to me!

**Pull request merged**